backend/lm_backend/api/schemas/license_server.py

```diff
@@ -23,39 +27,47 @@ class LicenseServerWithoutConfigIdCreateSchema(BaseCreateSchema):
     The config_id will be handled in the endpoint.
     """
 
-    host: str
-    port: PositiveInt
+    host: str = Field(..., title="Host", description="The host of the license server.")
```

Comment:
We should also make sure valid hostnames are supplied here. According to https://www.rfc-editor.org/rfc/rfc1123: [a-z0-9-.]+ (basically)

Reply:
> "Host software MUST handle host names of up to 63 characters and SHOULD handle host names of up to 255 characters."

This regex would consider the 63 characters limit: `"^(?!-)[A-Za-z0-9-]{1,63}(?<!-)(\.[A-Za-z0-9-]{1,63})*$"`